pkg/client/devops/jenkins/options.go

```diff
@@ -87,4 +93,7 @@ func (s *Options) AddFlags(fs *pflag.FlagSet, c *Options) {
 
 	fs.StringVar(&s.Namespace, "namespace", c.Namespace, "Namespace where devops system is in.")
 	fs.StringVar(&s.WorkerNamespace, "worker-namespace", c.WorkerNamespace, "Namespace where Jenkins agent workers are in.")
+	fs.DurationVar(&s.ReloadCasCPeriod, "reload-casc-period", c.ReloadCasCPeriod,
+		"ReloadCasCPeriod specifies the total duration that controller should delay the reload action for "+
+			"jenkins-casc-config ConfigMap change, and it is only valid for controller manager. Default: \"70s\"")
```

Comment:
you don't need to provide the default value in the description. see also https://github.com/jenkins-zh/jenkins-cli/blob/c1846ac042deec330a39dd161679bacaf1386b3a/app/cmd/job_build.go#L41

let's assume, someone changes the default value. then he needs to change two places instead of one

Reply:
lack of time unit description

Reply:
Alright.

Reply:
Option with duration type can be set to `1s`, `1m`, `1h`, or `1d`, so I think we can omit the time unit description.

Reply:
That's ok. But provide an example will be better. Such as